src/dlio/odom.cc

```diff
@@ -64,6 +64,7 @@ dlio::OdomNode::OdomNode(ros::NodeHandle node_handle) : nh(node_handle) {
   this->imu_meas.lin_accel[2] = 0.;
 
   this->imu_buffer.set_capacity(this->imu_buffer_size_);
+  this->pose_buffer.set_capacity(this->imu_buffer_size_);
```

Comment:
Buffer size is currently 5000. A seperate ros parameter could be made to change the size of this buffer